ChemistryLib/PhreeqcKernelData/CreateInitialAqueousSolution.cpp

```diff
@@ -62,6 +62,17 @@ InitialAqueousSolution createInitialAqueousSolution(
             "for the transport process of hydrogen.");
     }
 
+    auto const means_of_adjusting_charge_in_str =
+        //! \ogs_file_param{prj__chemical_system__solution__means_of_adjusting_charge}
+        config.getConfigParameter<std::string>("means_of_adjusting_charge", "");
```

Comment:
I find "means of something" rather ugly.
Just an idea:
```xml
<charge_adjusting_mechanism>pH</charge_adjusting_mechanism>
```

Reply:
How about `<charge_balance>pH</charge_balance>`?

Reply:
"charge_balance" much more better

Reply:
Improved.